docs/src/home.md

```diff
@@ -1,160 +1,270 @@
+ (which can find *all* steady states of systems with multiple ones), by forward ODE simulations using
+ [SteadyStateDiffEq.jl)](https://github.com/SciML/SteadyStateDiffEq.jl), or by nonlinear systems 
+ solving using [NonlinearSolve.jl](https://github.com/SciML/NonlinearSolve.jl).
+- [BifurcationKit.jl](https://github.com/bifurcationkit/BifurcationKit.jl) can be used to [compute 
+ bifurcation diagrams](@ref ref) of models' steady states (including finding periodic orbits).
+- [DynamicalSystems.jl](https://github.com/JuliaDynamics/DynamicalSystems.jl) can be used to compute 
+ model [basins of attraction](@ref ref) and [Lyapunov spectrums](@ref ref).
+- [StructuralIdentifiability.jl](https://github.com/SciML/StructuralIdentifiability.jl) can be used 
+ to [perform structural identifiability analysis](@ref ref).
+- [Optimization.jl](https://github.com/SciML/Optimization.jl), [DiffEqParamEstim.jl](https://github.com/SciML/DiffEqParamEstim.jl), 
+ and [PEtab.jl](https://github.com/sebapersson/PEtab.jl) can all be used to [fit model parameters to data](@ref ref).
+- [GlobalSensitivity.jl](https://github.com/SciML/GlobalSensitivity.jl) can be used to perform 
+ [global sensitivity analysis](@ref ref) of model behaviors.
+ 
+#### [Features of packages built upon Catalyst](@id doc_home_features_other_packages)
+- Catalyst [`ReactionSystem`](@ref)s can be [imported from SBML files](@ref ref) via
+ [SBMLImporter.jl](https://github.com/SciML/SBMLImporter.jl) and [SBMLToolkit.jl](https://github.com/SciML/SBMLToolkit.jl), 
+ and [from BioNetGen .net files](@ref ref) and various stoichiometric matrix network representations
+ using [ReactionNetworkImporters.jl](https://github.com/SciML/ReactionNetworkImporters.jl).
+- [MomentClosure.jl](https://github.com/augustinas1/MomentClosure.jl) allows generation of symbolic 
+ ModelingToolkit `ODESystem`s that represent moment closure approximations to moments of the 
+ Chemical Master Equation, from reaction networks defined in Catalyst.
+- [FiniteStateProjection.jl](https://github.com/kaandocal/FiniteStateProjection.jl)
+ allows the construction and numerical solution of Chemical Master Equation
+ models from reaction networks defined in Catalyst.
+- [DelaySSAToolkit.jl](https://github.com/palmtree2013/DelaySSAToolkit.jl) can
+ augment Catalyst reaction network models with delays, and can simulate the
+ resulting stochastic chemical kinetics with delays models.
+- [BondGraphs.jl](https://github.com/jedforrest/BondGraphs.jl), a package for
+ constructing and analyzing bond graphs models, which can take Catalyst models as input.
+- [PEtab.jl](https://github.com/sebapersson/PEtab.jl), a package that implements the PEtab format for 
+ fitting reaction network ODEs to data. Input can be provided either as SBML files or as Catalyst 
+ `ReactionSystem`s.
+
+## [How to read this documentation](@id doc_home_documentation)
```

Comment:
I figured something like this would be useful. I think there are really two crucial bits of information here:
- That the API exists. I think many do not know about it or what it is/the abbreviation mean. I think mentioning it, and what it is, early, can be useful.
- Mention that the code in the docs is run dynamically, but only sometimes are we actually displaying the output.